keps/sig-node/2403-pod-resources-allocatable-resources/README.md

```diff
@@ -193,6 +238,8 @@ Add additional tests to prove that unhealthy devices are skipped as part of GetA
 - [X] Allowing time for feedback (1 year).
 - [X] Risks have been addressed.
 
+#### Deprecation
+
```

Comment:
What goes here?

Reply:
(here and below) I should have marked this PR WIP, thanks for your review and for pointing it out. I'll have a non-WIP upload shortly.